plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/RocksDBKeyValueStorageFactory.java

```diff
@@ -44,6 +44,7 @@
 public class RocksDBKeyValueStorageFactory implements KeyValueStorageFactory {
 
   private static final Logger LOG = LogManager.getLogger();
+  /** Source of truth for the default database version. */
```

Comment:
How about deleting this comment instead? 😉

Reply:
It was there before precisely because it wasn't in a static final member so now that it is I'm good with that!